lib/panels/Contour/index.js

```diff
@@ -154,11 +154,11 @@ export default React.createClass({
                   onReset={ this.onReset }
                   needsApply={ needsApply }
                   toolbarButtons= {
-                    <i className='fa fa-fw fa-plus is-clickable' onClick={ this.addContour }></i>
+                    <i className={style.plus} onClick={ this.addContour }></i>
                   }>
                     { this.state.values.map( (v, idx) => {
-                        return <div key={ idx } className='ContourPanel_line'>
-                                    <i className='fa fa-fw fa-trash-o is-clickable'
+                        return <div key={ idx } className={style.line}>
+                                    <i className={style.trash}
```

Comment:
You could have name the button .deleteButton instead of .trash. 
Same for the plus above. Those name should describe the element goal not the current icon implementation.